### PR TITLE
{numlib}[GCC/13.3.0] OpenBLAS v0.3.27

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bwd-rockylinux-94 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.161.08, Python 3.9.18
See https://gist.github.com/SebastianAchilles/d7971e09c5b5b73c6832a1978ba8e43c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/bcf8495267488c3101ced3d876bf15de for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0207u28a.bear.cluster - Linux RHEL 8.6, x86_64, AMD EPYC 9554 64-Core Processor (zen4), Python 3.6.8
See https://gist.github.com/branfosj/580ca9603572a8802ebd9589b9864350 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/3c5bc0ea330c2fc1bfcc1d9ed80e3a1a for a full test report.

**Comment by SebastianAchilles** [comment]
https://github.com/easybuilders/easybuild-easyconfigs/pull/20621/commits/18c579770fd16c833f07802ca71746270a5f1d1f adds the patch from https://github.com/OpenMathLib/OpenBLAS/pull/4717

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-94 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.161.08, Python 3.9.18
See https://gist.github.com/SebastianAchilles/776eeaaa03f4f777da597c1bddb1f22f for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jscclxc1.int.jsc-clx.fz-juelich.de - Linux Rocky Linux 9.4, x86_64, Intel Xeon Processor (Cascadelake) (cascadelake), Python 3.9.18
See https://gist.github.com/SebastianAchilles/dd8b59d6b504a3f3a0b05adcb57ef6a0 for a full test report.

**Comment by bartoldeman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bartoldeman** [comment]
@boegelbot please test @ generoso

**Review by bartoldeman — APPROVED** [review]
lgtm

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login2.int.archimedes.c3.ca - Linux Rocky Linux 8.9 (Green Obsidian), x86_64, Intel Xeon Processor (Skylake, IBRS) (cascadelake), Python 3.11.4
See https://gist.github.com/bartoldeman/71bd973284a5bd96f9a151951dec597e for a full test report.

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.27-GCC-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name = 'OpenBLAS'
+version = '0.3.27'
+
+homepage = 'http://www.openblas.net/'
```
Comment (bartoldeman) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.27-GCC-13.3.0.eb:4]: can use https

**Review by bartoldeman — CHANGES_REQUESTED** [review]
tiny nitpick about http after all

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.27-GCC-13.3.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.27-GCC-13.3.0.eb:4]: Good catch, I am going to update the MR.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Review by branfosj — APPROVED** [review]

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrc0870.jureca - Linux Rocky Linux 8.9, AArch64, ARM UNKNOWN (neoverse_n1), 2 x NVIDIA NVIDIA A100-PCIE-40GB, 535.154.05, Python 3.6.8
See https://gist.github.com/SebastianAchilles/39e653f263acc67e7dfac00f1f6d6b0c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0207u28a.bear.cluster - Linux RHEL 8.6, x86_64, AMD EPYC 9554 64-Core Processor (zen4), Python 3.6.8
See https://gist.github.com/branfosj/bdb77cd056b528302601c0b8bce468a2 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/6f0292cc3a1f4c80affbdf2b1739d69a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/0470e8c6a90623e9b5a4608daca8c649 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0207u20a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (sapphirerapids), Python 3.6.8
See https://gist.github.com/branfosj/a3aa492ef9e5751ade50d94a0573d3e1 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
starfive - Linux Debian GNU/Linux n/a, RISC-V-64, UNKNOWN, Python 3.10.9
See https://gist.github.com/bedroge/10e59d8d2ce5e1836d19fb07476b550d for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
arriesgado-2 - Linux Ubuntu 22.04, RISC-V-64, UNKNOWN (u74mc), Python 3.10.12
See https://gist.github.com/SebastianAchilles/5972f3e02ecf9fd106d65f90069b0b34 for a full test report.

**Pull request merged**